message_definitions/v1.0/ras_a.xml

```diff
@@ -16,6 +16,36 @@
         <param index="2" label="X Position in the camera frame" minValue="0" increment="1">Position given by the number of pixels in X direction of the camera frame. Origin on the bottom left corner of the camera frame.</param>
         <param index="3" label="Y Position in the camera frame" minValue="0" increment="1">Position given by the number of pixels in Y direction of the camera frame. Origin on the bottom left corner of the camera frame.</param>
       </entry>
+      <entry value="36" name="MAV_CMD_DO_FOLLOW_SYS_ID" hasLocation="false" isDestination="false">
+        <description>Begin following a target</description>
+        <param index="1" label="System ID" minValue="1" maxValue="255" increment="1">System ID (of the FOLLOW_TARGET beacon).</param>
+        <param index="2">Reserved</param>
+        <param index="3">Reserved</param>
+        <param index="4" label="Altitude Mode" minValue="0" maxValue="2" increment="1">Altitude mode: 0: Keep current altitude, 1: keep altitude difference to target, 2: go to a fixed altitude above home.</param>
+        <param index="5" label="Altitude" units="m">Altitude above home. (used if mode=2)</param>
+        <param index="6">Reserved</param>
+        <param index="7" label="Time to Land" units="s" minValue="0">Time to land in which the MAV should go to the default position hold mode after a message RX timeout.</param>
+      </entry>
+      <entry value="37" name="MAV_CMD_DO_FOLLOW_POI" hasLocation="false" isDestination="false">
+        <description>Begin following a target specified by POI ID</description>
+        <param index="1" label="POI UID Upper 32 Bits">The upper 32 bits of the uid from a POI_REPORT.</param>
+        <param index="2" label="POI UID Lower 32 Bits">The lower 32 bits of the uid from a POI_REPORT</param>
+        <param index="3" label="Mode Flags" enum="FOLLOW_POI_MODE_FLAGS">Bitmask of option flags.</param>
+        <param index="4" label="Altitude" units="mm or degE7">Altitude above home. (used if mode=2)</param>
+        <param index="5" label="Azimuth" units="degE7">Altitude above home. (used if mode=2)</param>
+        <param index="6" label="Range" units="mm">Altitude above home. (used if mode=2)</param>
```

Comment:
This is different then my proposal, it seems who half kept what I suggested, but also trying to simplify it, just wanted ot make sure this wasn't an oversight, or maybe this is what I gave you and you didn't have the latest?

Reply:
that was inadvertent, sorry @thomas-watters-skydio

Reply:
@radarku @thomas-watters-skydio This looks right to me. It matches what I have in ras_a.xml right now. I just don't actually populate those fields yet. Maybe we were just working off different versions.

Reply:
@thomas-watters-skydio is the difference covered by 146caeed47f897519c9016885b9d8f7682c61b57 ?